app/Models/User.php

```diff
@@ -12,7 +12,7 @@ class User extends Authenticatable
 
     /**
      * The attributes that are mass assignable.
-     * 
+     *
      * @var array
      */
```

Comment:
I guess the bold developers of Laravel do not add the comment listing all attributes of this class. Could you add it?